tests/test_freqtradebot.py

```diff
@@ -2141,16 +2144,18 @@ def test_check_handle_timedout_exception(default_conf, ticker, mocker, caplog) -
         fee_open=0.0,
         fee_close=0.0,
         stake_amount=1,
-        open_date=arrow.utcnow().shift(minutes=-601).datetime,
+        open_date=open_date.datetime,
         is_open=True
     )
 
     Trade.session.add(trade_buy)
 
     freqtrade.check_handle_timedout()
-    assert log_has_re(r'Cannot query order for Trade\(id=1, pair=ETH/BTC, amount=90.99181073, '
-                      r'open_rate=0.00001099, open_since=10 hours ago\) due to Traceback \(most '
-                      r'recent call last\):\n.*', caplog)
+    assert log_has_re(f"Cannot query order for Trade\\(id=1, pair=ETH/BTC, amount=90.99181073, "
+                      f"open_rate=0.00001099, open_since="
+                      f"{open_date.strftime('%Y-%m-%d %H:%M:%S')} "
+                      f"\\(10 hours ago\\)\\) due to Traceback \\(most recent call last\\):\n*",
```

Comment:
you should be able to combine r-strings and f-strings - so only the 3rd line needs to be an f-string - avoiding most of the double `\\` escaping magic ...

Reply:
Done 👍